test/units/modules/cloud/docker/test_docker_network.py

```diff
@@ -0,0 +1,38 @@
+import sys
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
```

Comment:
I have just been told that new unit tests must use pytest and not unittest. (That should also solve the Python 2.6 problems...)

Reply:
I have never used pytest before, but that looks way more elegant